### PR TITLE
docs: :memo: Add workshop documentation and references

### PR DESCRIPTION
**Pull request by rmarting**
## Description

Add documentation and references from the set of workshops delivered between 27th, September and 5th, October.

## Why

Request by the team instead of the use of Google Slides deck.

## Issue

N/A

## Checklist

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [X] I have performed a self-review of my changes
- [X] I have commented my changes, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by rmarting** [comment]
@evegufy All the content was pushed here for your review. Includes the workshop's summaries and the PoC Helm Chart implementation. If you have comments or questions, please, let me know.

Thanks

File: charts/portal-iam/.helmignore
Comment (evegufy) [inline charts/portal-iam/.helmignore:1]: Could you please move the 'portal-iam' chart directory to /docs/consultation/ ?
As won't change in the short term to the new chart structure, it's inconvenient to have it in the charts dir as the chart releaser actions works with that directory.

**Review by evegufy — CHANGES_REQUESTED** [review]
Hi @rmarting the PR looks really good!

Could you please sign the Eclipse Contributor Agreement?:
https://www.eclipse.org/legal/ECA.php
You'd need to create a Eclipse account 
https://accounts.eclipse.org/user/register
and then sign the ECA here:
https://accounts.eclipse.org/user/eca

I also made a comment regarding the location of the portal-iam example chart and there's conflict with the consultation.md file, could you please solve that?

Thank you!

**Comment by evegufy** [comment]
@rmarting please also update the PR description, it's still from the 1rst workshop, thanks.

**Comment by rmarting** [comment]
@evegufy Moved the Helm Chart to the new location, however, I am trying to create my Eclipse Foundation account but it is not working, as I am not getting the confirmation email.

The PR is named without references for the first workshop, just the name of my personal branch. Are you asking to change the name of my branch?

Thanks

**Comment by evegufy** [comment]
> @evegufy Moved the Helm Chart to the new location, however, I am trying to create my Eclipse Foundation account but it is not working, as I am not getting the confirmation email.
> 
> The PR is named without references for the first workshop, just the name of my personal branch. Are you asking to change the name of my branch?
> 
> Thanks

Hi @rmarting thanks for moving the example chart!

Regarding updating the PR description, I just meant updating the main comment of the PR: 
image

Regarding the missing confirmation when creating of the Eclipse Foundation account: That's a bit out of my reach but Ican create a EF Helpdesk issue, I assume you used you RH mail address for creating the account, right?

There's still the conflict in the consultation.md file, could you please solve that?

**Comment by rmarting** [comment]
Sorry @evegufy as I can see the conflict in the `consultation.md` file, as in the PR that file is safe to be merged. Could you give more context about which conflicts are there?

Updated the PR description, and I am still waiting to the the Eclipse Foundation account. Tested with my RH email, and personal one ... but same results! :disappointed:

**Comment by evegufy** [comment]
> Sorry @evegufy as I can see the conflict in the `consultation.md` file, as in the PR that file is safe to be merged. Could you give more context about which conflicts are there?
> 
> Updated the PR description, and I am still waiting to the the Eclipse Foundation account. Tested with my RH email, and personal one ... but same results! 😞

@rmarting I'm not sure, I understand your comment correctly but I mean the following conflict:
image
I recommend rebasing to the 'docs/consultation' base branch.

Regarding Eclipse Foundation Account: I'll check how to support and get back to you.

**Comment by rmarting** [comment]
Done! Just waiting to create the EF account property. The main issue is that the confirmation email do not arrive on time, it is taking several hours, and when I click in the links to confirm, it is expired. I am trying from time to time, but it is always the same issue. I will confirm you when my account is ready (as, this is the only thing pending to do)

**Comment by rmarting** [comment]
@evegufy Signed my ECA and now all the checks have passed! So, DONE! :tada: :sparkles:

**Pull request merged**